src/App.js

```diff
@@ -20,15 +28,19 @@ export default class App extends React.Component {
   }
   render() {
     return (
-      <div id="requiredContainer">
-        <Header />
-        { this.state.data
-        ? <PokeList data={this.state.data}/>
-        : 'loading content' 
-        } 
-      </div>
+      <Router>
+        <div>
+          <Header />
+          <Link to="/">go home</Link>
+          <Link to="./AboutMe">about me</Link>
+          <Switch>
+            <Route exact path="/:pokemon?" component={PokeList} />
+            <Route exact path="/data/:pokeId" component={Detail} />
```

Comment:
so to get this detail card to route properly, the route should be <Route exact path="/pokemon/:pokeId" component={Detail} /> because it needs to follow the same syntax that you have on the detail page that maps this for you.